### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by ericvanvoorthuizen** [robvdpol/RaceControl#399]
**Describe the bug**
When start the internal player an error occured.
HRESULT: [0x887A0005], Message: [The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.

**To Reproduce**
1. Play any stream with the Watch button
2. See RaceControl.log

**Expected behavior**
Play stream with internal player

**Screenshots**
<img width="301" alt="RC_Err" src="https://user-images.githubusercontent.com/26880302/160247488-92a3461a-b8a3-498e-a20d-95cc26d139ea.PNG">

**Desktop (please complete the following information):**
see MC.txt

**F1TV account**
- Subscription type  = Pro
- Country = NL
- Using VPN or proxy? No

**Additional context**
[MC.txt](https://github.com/robvdpol/RaceControl/files/8355809/MC.txt)
[RaceControl.log](https://github.com/robvdpol/RaceControl/files/8355810/RaceControl.log)

**Comment by Spexxel** [comment]
Is this an issue since one of the recent updates (has it worked before) or is this the first time using Race Control?

**Comment by ericvanvoorthuizen** [comment]
> 

used RaceControl 2 weeks ago for the first time on this device with 2.6.1 and on 2.6.2 issue is also present.
Don't tested it with older versions though.

**Comment by SuRGeoNix** [comment]
It looks like a GPU(there are two of them, might uses the Intel instead of Philips)/DirectX issue. Can you provide a Flyleaf.log if you have one?

**Comment by Spexxel** [comment]
There is only one GPU as far I could see in the MC.log. The philips one is the monitor

```
Graphics
		Monitor
			Name	Philips FTV on Intel HD Graphics
			Current Resolution	1280x720 pixels
			Work Resolution	1280x680 pixels
			State	Enabled, Primary
			Monitor Width	1920
			Monitor Height	1080
			Monitor BPP	32 bits per pixel
			Monitor Frequency	60 Hz
			Device	\\.\DISPLAY1\Monitor0
		Intel HD Graphics
			Manufacturer	Intel
			Model	HD Graphics
			Device ID	8086-0046
			Revision	19
			Subvendor	ASRock (1849)
			Current Performance Level	Level 0
			Driver version	8.15.10.2900
				Count of performance levels : 1
					Level 1 - "Perf Level 0"
					```

**Comment by SuRGeoNix** [comment]
True, my mistake. Do you have issue with other players? Can you try update your GPU drivers?

**Comment by Spexxel** [comment]
If I am to believe the MC.log there are also quite a lot on Windows updates which failed to install. Might just try and update that too while you are at it

**Comment by ericvanvoorthuizen** [comment]
No issues with VLC, MPV players within RaceControl.

Windows is fully updated.
Windows update errors are just history when update failed for some reason.

Now installing 21H2, just to be sure.


Op za 26 mrt. 2022 20:19 schreef Khaak ***@***.***>:

**Comment by ericvanvoorthuizen** [comment]
On Windows 10 21H2 same issue is occuring

**Comment by SuRGeoNix** [comment]
Sorry for the late replay, this is an old GPU and unfortunately will not be supported by the internal player. It is possible that it will work by improving the fall back to WARP device but it will still have some issues. You can find more details [here](https://github.com/SuRGeoNix/Flyleaf/issues/157)
